src/test/resources/pages/instructorFeedbackAllSessionTypes.html

```diff
@@ -6,7 +6,7 @@
   </h1>
   <br>
   <div class="well well-plain">
-    <form action="/page/instructorFeedbackAdd" class="form-group" id="form_feedbacksession" method="post">
+    <form action="/page/instructorFeedbackAdd" class="form-group" data-editsessiondetails="" id="form_feedbacksession" method="post">
```

Comment:
Looking at the Travis error logs, yes, they do. God mode is just finishing up on my computer. Will commit the updated HTML files soon.